nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -239,7 +239,106 @@ providers:
         networks:
           - Public Internet
           - Private Network (10.0.0.0/8 only)
-        labels: *provider_limestone_pools_s1_small_labels
```

Comment:
Hmm, this is pretty large change, just for a single flavor. I would prefer is we could come up with another way using yaml anchors.

Reply:
How would you do that?